redbot/cogs/trivia/session.py

```diff
@@ -282,41 +282,47 @@ def force_stop(self):
         LOG.debug("Force stopping trivia session; #%s in %s", channel, channel.guild.id)
 
     async def pay_winner(self, multiplier: float):
-        """Pay the winner of this trivia session.
+        """Pay the winner(s) of this trivia session.
 
-        The winner is only payed if there are at least 3 human contestants.
+        Payout only occurs if there are at least 3 human contestants.
+        If a tie occurs the payout is split evenly among the winners.
 
         Parameters
         ----------
         multiplier : float
-            The coefficient of the winner's score, used to determine the amount
+            The coefficient of the winning score, used to determine the amount
             paid.
 
         """
-        (winner, score) = next((tup for tup in self.scores.most_common(1)), (None, None))
-        me_ = self.ctx.guild.me
-        if winner is not None and winner != me_ and score > 0:
-            contestants = list(self.scores.keys())
-            if me_ in contestants:
-                contestants.remove(me_)
-            if len(contestants) >= 3:
-                amount = int(multiplier * score)
-                if amount > 0:
-                    LOG.debug("Paying trivia winner: %d credits --> %s", amount, str(winner))
-                    try:
-                        await bank.deposit_credits(winner, int(multiplier * score))
-                    except errors.BalanceTooHigh as e:
-                        await bank.set_balance(winner, e.max_balance)
-                    await self.ctx.send(
-                        _(
-                            "Congratulations, {user}, you have received {num} {currency}"
-                            " for coming first."
-                        ).format(
-                            user=winner.display_name,
-                            num=humanize_number(amount),
-                            currency=await bank.get_currency_name(self.ctx.guild),
-                        )
-                    )
+        if not self.scores:
+            return
+        top_score = self.scores.most_common(1)[0][1]
+        winners = []
+        num_humans = 0
+        for (player, score) in self.scores.items():
+            if not player.bot:
+                if score == top_score:
+                    winners.append(player)
+                num_humans += 1
+        if len(winners) == 0 or num_humans < 3:
```

Comment:
According to our style guide, the fact that empty sequences evaluate to false should be used instead of `len()` (you can see PEP 8 for more information)
```suggestion
        if not winners or num_humans < 3:
```